azdev/config/cli.flake8

```diff
@@ -2,13 +2,20 @@
 max-line-length = 120
 max-complexity = 10
 ignore =
-    E501,  # line too long, it is covered by pylint
-    E722,  # bare except, bad practice, to be removed in the future
-    F401,  # imported but unused, too many violations, to be removed in the future
-    F811,  # redefinition of unused, to be removed in the future
-    C901   # code flow is too complex, too many violations, to be removed in the future
-    W503   # line break before binary operator
-    W504   # line break after binary operator effect on readability is subjective
+    # line too long, it is covered by pylint
+    E501,
```

Comment:
Any reason for reformatting?

Reply:
When `pylint==2.11.1`, it installs `mccabe` 0.6.1. `flake8` also requires `mccabe`, and only **4.0.1** meet the [version constraint](https://github.com/PyCQA/flake8/blob/82b698e09996cdde5d473e234681d8380810d7a2/setup.cfg#L42).
```
  - pylint [required: ==2.11.1, installed: 2.11.1]
    - astroid [required: >=2.8.0,<2.9, installed: 2.8.6]
      - lazy-object-proxy [required: >=1.4.0, installed: 1.9.0]
      - setuptools [required: >=20.0, installed: 63.2.0]
      - wrapt [required: >=1.11,<1.14, installed: 1.13.3]
    - isort [required: >=4.2.5,<6, installed: 5.12.0]
    - mccabe [required: >=0.6,<0.7, installed: 0.6.1]
    - platformdirs [required: >=2.2.0, installed: 2.6.2]
    - toml [required: >=0.7.1, installed: 0.10.2]
 
- flake8 [required: Any, installed: 4.0.1]
    - mccabe [required: >=0.6.0,<0.7.0, installed: 0.6.1]
    - pycodestyle [required: >=2.8.0,<2.9.0, installed: 2.8.0]
    - pyflakes [required: >=2.4.0,<2.5.0, installed: 2.4.0]
```
After upgrading PyLint 2.7, `mccabe` becomes [required: >=0.6,<0.8, installed: 0.7.0], and flake8 becomes **6.0.0**.

Reply:
Does this issue still exist in pylint 2.17.4? https://pypi.org/project/pylint/

Reply:
When run `azdev style`, it uses `pylintrc` and `.flake8` in CLI/Extension repo folder. If the files do not exist, it use `cli.xxxx` or `ext.xxx` in `azdev config` folder.
CLI's `.flake8` is updated in https://github.com/Azure/azure-cli/pull/25370.